src/components/Card.astro

```diff
@@ -0,0 +1,63 @@
+---
+export interface Props {
```

Comment:
I think this is only used by the pages Astro put in by default. We should get rid of those (and then of this), and just leave the pages we're working on ourselves.